storm-core/test/clj/org/apache/storm/nimbus_test.clj

```diff
@@ -30,6 +30,7 @@
   (:import [java.util HashMap])
   (:import [java.io File])
   (:import [org.apache.storm.utils Time Utils])
+  (:import [org.apache.storm.zookeeper Zookeeper])
```

Comment:
Again this is for stubbing, so if you want some help with this please let me know.